apps/plasmo/src/components/DateWithEvents.tsx

```
@@ -0,0 +1,109 @@
+      "२": "2",
+      "३": "3",
+      "४": "4",
+      "५": "5",
+      "६": "6",
+      "७": "7",
+      "८": "8",
+      "९": "9"
+    }
+    return parseInt(
+      nepaliNum
+        .split("")
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)